db/schema.rb

```diff
@@ -11,55 +11,52 @@
 #
 # It's strongly recommended that you check this file into your version control system.
 
-ActiveRecord::Schema.define(version: 20_160_907_024_814) do
+ActiveRecord::Schema.define(version: 20160913185739) do
+
   # These are extensions that must be enabled in order to support this database
-  enable_extension 'plpgsql'
+  enable_extension "plpgsql"
 
-  create_table 'games', force: :cascade do |t|
-    t.integer  'game_id'
-    t.string   'name'
-    t.datetime 'created_at', null: false
-    t.datetime 'updated_at', null: false
+  create_table "games", force: :cascade do |t|
+    t.string   "name"
+    t.datetime "created_at", null: false
+    t.datetime "updated_at", null: false
   end
 
-  create_table 'pieces', force: :cascade do |t|
-    t.integer  'piece_id'
-    t.integer  'player_id'
-    t.string   'color'
-    t.integer  'game_id'
-    t.datetime 'created_at', null: false
-    t.datetime 'updated_at', null: false
-    t.string   'type'
-    t.integer  'x_position'
-    t.integer  'y_position'
+  create_table "pieces", force: :cascade do |t|
+    t.integer  "player_id"
+    t.string   "color"
+    t.integer  "game_id"
+    t.datetime "created_at", null: false
+    t.datetime "updated_at", null: false
+    t.string   "type"
+    t.integer  "x_position"
+    t.integer  "y_position"
   end
 
-  create_table 'players', force: :cascade do |t|
-    t.integer  'player_id'
-    t.integer  'user_id'
-    t.string   'color'
-    t.integer  'game_id'
-    t.datetime 'created_at', null: false
-    t.datetime 'updated_at', null: false
+  create_table "players", force: :cascade do |t|
+    t.integer  "user_id"
+    t.string   "color"
+    t.integer  "game_id"
+    t.datetime "created_at", null: false
+    t.datetime "updated_at", null: false
   end
 
-  create_table 'users', force: :cascade do |t|
-    t.string   'email',                  default: '', null: false
-    t.string   'encrypted_password',     default: '', null: false
-    t.string   'reset_password_token'
-    t.datetime 'reset_password_sent_at'
-    t.datetime 'remember_created_at'
-    t.integer  'sign_in_count', default: 0, null: false
-    t.datetime 'current_sign_in_at'
-    t.datetime 'last_sign_in_at'
-    t.inet     'current_sign_in_ip'
-    t.inet     'last_sign_in_ip'
-    t.datetime 'created_at',                          null: false
-    t.datetime 'updated_at',                          null: false
-    t.string   'provider'
```

Comment:
The provider id and uid columns are necessary for oauth, but i am wondering how its here in your branch @wdehaes , because the oauth branch where i have added the new fields(provider id and uid) is yet to be merged with master. In that case, i feel @erniejamison would have added them while adding some feature.